Running ng-add on new project fails

<!--
PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.

ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
-->

## I'm submitting a...

<!-- Check one of the following options with "x" -->
<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

ng add will fail if `assets/svg` folder not exists

## Expected behavior

ng add should not run `npm run generate-icons` if there is no svg folder / shouldn't run at all / console log instructions with next steps

## Minimal reproduction of the problem with instructions

1. `ng new svg-icons-test`
2. `ng add @ngneat/svg-icon`
3. Output:
```
ng add @ngneat/svg-icon
Installing packages for tooling via npm.
Installed packages for tooling via npm.
    ✅️ Update package.json
    🔍 Installing package...
    ✅️ @ngneat/svg-icon is imported
    👏 @ngneat/svg-icon ready to use
UPDATE src/app/app.module.ts (459 bytes)
✔ Packages installed successfully.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! svg-icons-test@0.0.0 generate-icons: `svg-to-ts`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the svg-icons-test@0.0.0 generate-icons script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vladta/.npm/_logs/2021-01-19T09_32_51_893Z-debug.log
Unknown error: Error: Command failed: npm run generate-icons
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! svg-icons-test@0.0.0 generate-icons: `svg-to-ts`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the svg-icons-test@0.0.0 generate-icons script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vladta/.npm/_logs/2021-01-19T09_32_51_893Z-debug.log
```

## What is the motivation / use case for changing the behavior?

<!-- Describe the motivation or the concrete use case. -->

## Environment

<pre><code>
Angular version: X.Y.Z
<!-- Check whether this is still an issue in the most recent Angular version -->

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  <!-- run `node --version` -->
- Platform:  <!-- Mac, Linux, Windows -->

Others:
<!-- Anything else relevant?  Operating system version, IDE, package manager, HTTP server, ... -->
</code></pre>

--- a/schematics/src/ng-add/index.ts
+++ b/schematics/src/ng-add/index.ts
@@ -120,8 +120,17 @@ function addScripts({ project }): Rule {
 }
 
 process.on('exit', function() {
+  const dirPath = './src/assets/svg';
+  if(!fs.existsSync(dirPath)){
+    fs.mkdirSync(dirPath);
+  }
+  const dir = fs.readdirSync(dirPath);
+  if(dir.length < 1){
+    console.log(`✅️ Ready. Put your svg icons in src/assets/svg and then 'npm run generate-icons'`);
+  }
   require('child_process').execSync('npm run generate-icons');
   console.log(`✅️ npm run generate-icons`);
+  return;
 });
 
 export default function ngAdd(options: Schema): Rule {

